src/tasks/package.js

```diff
@@ -1,7 +1,10 @@
 /* eslint-disable no-template-curly-in-string */
+const path = require('path');
+const meta = require('user-meta');
+const gitUsername = require('git-username');
 const { json, install } = require('mrm-core');
 
-const packages = ['schema-utils'];
+const packages = ['schema-utils', 'loader-utils'];
```

Comment:
ℹ️  This makes the `package` tasks loader specific and needs to be handle by the e.g upcoming CLI based on a flag e.g 

```
$ webpack-defaults new --loader
$ webpack-defaults update --loader
$ webpack-defaults upgrade --loader
```

Reply:
Yeah, I'm working on a CLI wrapper that will essentially give you an option to pick ..

- loader
- plugin
  - Initial
  - Upgrade
  - You get the point

Given that at the moment, the vast majority of what we contend with is loaders if figured what the hell. I'll pull it & add it back once you can pick your path.

Reply:
Yep, in the short-term we can aswell  leave it, `loader-utils` isn't fully accurate anyways since it can be used in a generic way in `webpack` e.g `stringifyRequest, interpolate` etc :)